doc/classes/EditorImportPlugin.xml

```diff
@@ -241,7 +241,78 @@
 			<param index="2" name="custom_importer" type="String" default="&quot;&quot;" />
 			<param index="3" name="generator_parameters" type="Variant" default="null" />
 			<description>
-				This function can only be called during the [method _import] callback and it allows manually importing resources from it. This is useful when the imported file generates external resources that require importing (as example, images). Custom parameters for the ".import" file can be passed via the [param custom_options]. Additionally, in cases where multiple importers can handle a file, the [param custom_importer] can be specified to force a specific one. This function performs a resource import and returns immediately with a success or error code. [param generator_parameters] defines optional extra metadata which will be stored as [code skip-lint]generator_parameters[/code] in the [code]remap[/code] section of the [code].import[/code] file, for example to store a md5 hash of the source data.
+				This function can only be called during the [method _import] callback and it allows manually importing resources from it. This is useful when the imported file generates external resources that require importing (as example, images). Custom parameters for the ".import" file can be passed via the [param custom_options]. Additionally, in cases where multiple importers can handle a file, the [param custom_importer] can be specified to force a specific one. This function performs a resource import and returns immediately with a success or error code. [param generator_parameters] defines optional extra metadata which will be stored as [code skip-lint]generator_parameters[/code] in the [code]remap[/code] section of the [code].import[/code] file, for example to store a md5 hash of the source data. For example:
+				[codeblocks]
+				[gdscript]
+				func _import(source_file, save_path, options, r_platform_variants, r_gen_files):
+				    # Make textures folder if required.
+				    var source_file_folder: String = source_file.rsplit("/", true, 1)[0]
```

Comment:
```suggestion
				    var source_file_folder = source_file.rsplit("/", true, 1)[0]
```
We [don't add](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#dynamic-vs-static-typing) static typing to the documentation, for all